### PR TITLE
Add a PoseGraphModel RFC.

### PR DESCRIPTION
**Pull request by pifon2a**
[rendered](https://github.com/pifon2a/rfcs/blob/7fde14739cfc85103f772a72eacfee68125be3ff/text/0022-pose-graph-data.md)

### CODE REVIEW

File: text/0000-pose-graph-model.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+  std::set<int> finished_trajectories_;
+  std::set<int> frozen_trajectories_;
+  std::vector<Constraint> constraints_;
+};
+```
+
+## Discussion Points
+[discussion]: #discussion
+
+What is the preferred name: `PoseGraphModel` or `PoseGraphData` or `PoseGraphState`?
```
Comment (gaschler) [inline text/0000-pose-graph-model.md]: I like `PoseGraphModel` and `PoseGraphController`.

File: text/0000-pose-graph-model.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
+# Pose Graph Refactoring 1: Pose Graph Model
```
Comment (gaschler) [inline text/0000-pose-graph-model.md]: Feel free to upload the next work-in-progress RFCs when you have drafts so it's easier to see the big picture.

**Review by gaschler — APPROVED** [review]
Cool stuff!

File: text/0000-pose-graph-model.md (hunk 2 above)
Comment (pifon2a) [reply text/0000-pose-graph-model.md]: Sure, I will upload soon the `PoseGraphController` part.

**Comment by MichaelGrupp** [comment]
Voting for `PoseGraphState`.

**Pull request merged**